src/components/ContentPreview/ContentPreview.js

```diff
@@ -425,22 +437,37 @@ class ContentPreview extends PureComponent<Props, State> {
     }
 
     /**
-     * Event handler 'preview_metric' which adds in the file fetch time
+     * Event handler 'preview_metric' which also adds in the file fetch time if it's a load event
      *
-     * @param {Object} previewLoadMetrics - the object emitted by 'preview_metric'
+     * @param {Object} previewMetrics - the object emitted by 'preview_metric'
      * @return {void}
      */
-    onPreviewMetric = (previewLoadMetrics: PreviewLoadMetrics): void => {
+    onPreviewMetric = (previewMetrics: PreviewMetrics): void => {
         const { onMetric }: Props = this.props;
-        const totalFetchFileTime = this.getTotalFileFetchTime();
+        const { event_name } = previewMetrics;
+        let metrics = {
+            ...previewMetrics
+        };
 
         // We need to add in the total file fetch time to the file_info_time and value (total)
         // as preview does not do the files call
-        onMetric({
-            ...previewLoadMetrics,
-            file_info_time: totalFetchFileTime,
-            value: (previewLoadMetrics.value || 0) + totalFetchFileTime
-        });
+        if (event_name === 'load') {
+            const totalFetchFileTime = this.getTotalFileFetchTime();
+            const totalTime = (previewMetrics.value || 0) + totalFetchFileTime;
+
+            // If an unnatural fetch time occurs or is invalid, don't log the event
+            if (!totalFetchFileTime || !totalTime) {
```

Comment:
Is it possible for ```totalFetchFileTime``` to be 0? What is the harm of emitting the event if the fetch time is 0?

Reply:
the Elements cache is in memory instead of the browser cache. I could see it being 0 depending on how performance.now() rounds

Reply:
It shouldn't be possible for it to take 0 milliseconds to grab file info, even from browser cache. If it is 0, I would assume that something went wrong: when getting it from the cache OR requesting it via the API

Reply:
Can you quickly verify that when navigating between previews that the time is never 0 when the item is in the cache? Cache, meaning local cache (aka Cache.js), not browser cache?

Reply:
@JustinHoldstock any update on this?

Reply:
I've confirmed that we can have 0 ms values when a file is cached. We may need to rethink strategy on reducing redundant load time calls OR just begin filtering out total load times of 0 when calculating our load metrics (from our analytics queries). I'm leaning towards the latter, and removing this guard